### PR TITLE
*: remove driver usage in test.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: tidb_test.go [hunk 1]
```diff
@@ -239,63 +194,30 @@ func (s *testMainSuite) TestCaseInsensitive(c *C) {
...
-	_, err = db.Exec("create table t (c int)")
-	c.Assert(err, IsNil)
-	_, err = db.Exec("insert into t values (1), (2), (3)")
+	store := newStore(c, s.dbName)
+	se := newSession(c, store, s.dbName)
+	defer store.Close()
+	_, err := exec(c, se, "create table t (c int)")
 	c.Assert(err, IsNil)
-	_, err = db.Query("delete from `t` where `c` = ?", 1)
+	_, err = exec(c, se, "insert into t values (1), (2), (3)")
```
Comment (shenli) [inline tidb_test.go]: Use mustExecSQL?

Comment (qiuyesuifeng) [inline tidb_test.go]: +1

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**